index.js

```diff
@@ -166,6 +165,9 @@ WebtaskProfile.prototype.createToken = Bluebird.method(function (options, cb) {
         params.dr = 1;
     if (options.name)
         params.jtn = options.name;
+
+    if (options.auth0)
+        params.auth = options.auth0;
```

Comment:
This is going to result in a webtask token that contains the Auth0 client secret in plaintext. The Auth0 client secret must be stored as part of ectx.